third_party/build/resample/Makefile

```diff
@@ -44,6 +44,7 @@ ifneq ($(PJ_SHARED_LIBRARIES),)
 else
 ifneq ($(PJ_RESAMPLE_DLL),)
 export RESAMPLE_CFLAGS := -fPIC $(RESAMPLE_CFLAGS)
+export RESAMPLE_LDFLAGS := $(RESAMPLE_LDFLAGS)
```

Comment:
Instead of modifying the configure script, perhaps adding
```
export RESAMPLE_LDFLAGS := $(_LDFLAGS)
```
after https://github.com/pjsip/pjproject/blob/ae8f9796a45e95a945195d807c75868f0dd1607c/third_party/build/resample/Makefile#L22 is sufficient?

For any approach, we need to make sure it does not cause build failure to other configs, such as `--enable-shared` and `--enable-static`.

Reply:
If we use
```
export RESAMPLE_LDFLAGS := $(_LDFLAGS)
```
Then, all the library that is not needed on building libresample will be included. e.g: `-luuid -lm -lrt -lpthread  -lasound`

Reply:
From 
https://stackoverflow.com/questions/13984180/is-an-executable-built-differently-if-linked-against-a-library-thats-not-used

states that linking against a a static library not used should not have any effect. 
Using 
```
export RESAMPLE_LDFLAGS := $(_LDFLAGS)
```
should be sufficient.

Reply:
Tested here using `--enable-static`, `--enable-shared`, `--enable-resample-dll`, with 
```
export RESAMPLE_LDFLAGS := $(_LDFLAGS)
```
All succees

Reply:
> From https://stackoverflow.com/questions/13984180/is-an-executable-built-differently-if-linked-against-a-library-thats-not-used
> 
> states that linking against a a static library not used should not have any effect. Using

How about linked against dynamic library, will the resample requires the unneeded libraries as its dependencies (such as `-luuid -lm -lrt -lpthread -lasound`)?

Btw, just a note, after a quick check on other third-party Makefile's, some of them include the `*_LDFLAGS` line, but few others not.

Reply:
I check the file size of `libresample.so` and there's no file size difference when using the patch/not. 
Check with `readelf` tools and also there's no difference.